go.mod

```diff
@@ -3,14 +3,14 @@ module github.com/container-storage-interface/spec
 go 1.18
 
 require (
-	github.com/golang/protobuf v1.5.3
 	google.golang.org/grpc v1.57.0
+	google.golang.org/protobuf v1.31.0
```

Comment:
protoc-gen-go 1.32 is compiled below, but here the requirement is 1.31. can we standardize on 1.32?

Reply:
OK, fixed. I forgot to update this when I update the PR.